### ISSUE TITLE
Caught exception, RuntimeError: wrapped C/C++ object of type QTableView has been deleted

### ISSUE THREAD

**Issue by aderchox** [simgunz/anki21-addons_minimize-to-tray#4]
```

Debug info:
Anki 2.1.35 (84dcaa86) Python 3.8.0 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2021-03-13 12:45:40

Caught exception:
Traceback (most recent call last):
  File "C:\Users\xxx\AppData\Roaming\Anki2\addons21\85158043\__init__.py", line 47, in onActivated
    self.showAll()
  File "C:\Users\xxx\AppData\Roaming\Anki2\addons21\85158043\__init__.py", line 65, in showAll
    self.lastFocusedWidget.raise_()
RuntimeError: wrapped C/C++ object of type QTableView has been deleted

```

**Comment by simgunz** [comment]
Thanks for the report. I'll look into this issue soon.

**Comment by simgunz** [comment]
I have tried to reproduce it both in anki 2.1.35 and 2.1.44 and I couldn't.

Do you still face this problem with newer versions of Anki?
In this case, can you provide the exact steps to reproduce the error?

**Comment by simgunz** [comment]
Can you also try to remove and then redownload the addon?

**Comment by simgunz** [comment]
Fixed by d3d78f3
